tests/v2/test_waku_rln_relay_onchain.nim

```diff
@@ -104,6 +106,31 @@ proc createEthAccount(): Future[(keys.PrivateKey, Address)] {.async.} =
   return (pk, acc)
 
 procSuite "Waku-rln-relay":
+
+  ########################
+  ## Ganache installation
+  ########################
+
+  # We install Ganache
+  let installGanache = startProcess("npm", args = ["install",  "ganache"], options = {poUsePath})
+  debug "Ganache installation completed. Printing install log", returnCode=installGanache.waitForExit(), log=installGanache.outputstream.readAll()
```

Comment:
Chronicles at compile time remove all the statements below the `-d:chronicles_log_level=<level>` log level. So, in this case, if we set the log level to `INFO`, for example, it won't call `waitForExit()` nor `readAll()`.

```suggestion
  let installGanache = startProcess("npm", args = ["install",  "ganache"], options = {poUsePath})
  
  let installGanacheRc = installGanache.waitForExit()
  let installGanacheOutStream = installGanache.outputstream.readAll()
  debug "Ganache installation completed. Printing install log", returnCode=installGanacheRc, log=installGanacheOutStream
```